apps/web/src/pages/ApplicantDashboardPage/ApplicantDashboardPage.tsx

```diff
@@ -43,6 +134,227 @@ export const DashboardPage = ({ currentUser }: DashboardPageProps) => {
+                          >
+                            {communityInterestsQuery.length ? (
+                              <PreviewList.Root>
+                                {communityInterestsQuery.map(
+                                  (communityInterest) => (
+                                    <FunctionalCommunityListItem
+                                      key={communityInterest.id}
+                                      functionalCommunityListItemQuery={
+                                        communityInterest
+                                      }
+                                      headingAs="h4"
+                                    />
+                                  ),
+                                )}
+                              </PreviewList.Root>
+                            ) : (
+                              <Well data-h2-text-align="base(center)">
+                                <p data-h2-font-weight="base(bold)">
+                                  {intl.formatMessage({
+                                    defaultMessage:
+                                      "You haven't opted into any functional communities.",
+                                    id: "rrqAZ6",
+                                    description:
+                                      "Title for notice when there are no functional communities a user is a part of",
+                                  })}
+                                </p>
+                                <p>
+                                  {intl.formatMessage({
+                                    defaultMessage:
+                                      'Communities might be suggested based on your career experience. You can also add functional communities using the "Add a community" button.',
+                                    id: "w6JVdk",
+                                    description:
+                                      "Body for notice when there are no functional communities a user is a part of",
+                                  })}
+                                </p>
+                              </Well>
+                            )}
+                          </div>
+                        </Accordion.Content>
+                      </Accordion.Item>
+                    </Accordion.Root>
+                  </TaskCard.Item>
+                </TaskCard.Root>
+              </div>
+            ) : null}
+            <div
+              data-h2-display="base(flex)"
+              data-h2-flex-direction="base(column)"
+              data-h2-gap="base(x1)"
+              data-h2-max-width="p-tablet(x14)"
+            >
+              <ResourceBlock.Root
+                headingColor="tertiary"
+                headingAs="h2"
+                title={intl.formatMessage({
+                  defaultMessage: "Resources",
+                  id: "nGSUzp",
+                  description: "Card title for a 'resources' card",
+                })}
+              >
```

Comment:
Seems to be needed at some point, so I added it in with the bonus effect of helping page layout